presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -1288,6 +1271,24 @@ private void testInsertPartitionedBucketedTableWithUnionAll(HiveStorageFormat st
         assertFalse(getQueryRunner().tableExists(getSession(), tableName));
     }
 
+    @Test
+    public void testInsertTwiceToSamePartitionedBucket()
+    {
+        String tableName = "test_insert_twice_to_same_partitioned_bucket";
+        createPartitionedBucketedTable(tableName, HiveStorageFormat.RCBINARY);
+
+        String values = "1, 2, 'comment', 'F'";
```

Comment:
Inline. Per existing convention, we don't feel bad when mentioning same values twice.

Rationale is: when you repeat same values twice, you don't explicitly say the content is the same, but you can use string literals instead of `+` of `format`, which overall increases test readability.